src/FeatureBrowser/Cli/GenerateCommand.php

```diff
@@ -288,7 +288,7 @@ protected function makeOutputDir($directory)
         $path = $this->outputDirectory . $directory;
         if(!is_dir($path))
         {
-            mkdir($path, null, true);
+            mkdir($path, 0766, true);
```

Comment:
Just went with the default in the end (`0777`). If people want different permissions, they can set up the directory manually.